client/ayon_maya/plugins/load/load_ornatrix_alembic.py

```diff
@@ -0,0 +1,123 @@
+from maya import cmds
+
+from ayon_core.settings import get_project_settings
+from ayon_maya.api import plugin, lib
+from ayon_maya.api.pipeline import containerise
+from ayon_maya.api.lib import maintained_selection, unique_namespace
+from ayon_maya.api.plugin import get_load_color_for_product_type
+from ayon_core.lib import EnumDef
+
+
+class OxAlembicLoader(plugin.Loader):
+    """Ornatrix Alembic Loader"""
+
+    product_types = {"oxcache"}
+    representations = {"abc"}
+
+    label = "Ornatrix Alembic Loader"
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    @classmethod
+    def get_options(cls, contexts):
+        return [
+            EnumDef(
+                "import_options",
+                label="Import Options for Ornatrix Cache",
+                items={
+                    0: "Hair",
+                    1: "Guide"
+                },
+                default=0
+            )
+        ]
+
+    def load(self, context, name, namespace, options):
+        cmds.loadPlugin("Ornatrix", quiet=True)
+        # Build namespace
+        folder_name = context["folder"]["name"]
+        if namespace is None:
+            namespace = self.create_namespace(folder_name)
+
+        path = self.filepath_from_context(context)
+        path = path.replace("\\", "/")
+
+        ox_import_options = "; importAs={}".format(
+            options.get("import_options"))
+        group_name = "{}:{}".format(namespace, name)
+        project_name = context["project"]["name"]
+
+        with maintained_selection():
+            nodes = cmds.file(
+                path,
+                i=True,
+                type="Ornatrix Alembic Import",
+                namespace=namespace,
+                returnNewNodes=True,
+                groupName=group_name,
+                options=ox_import_options
+            )
+            nodes = cmds.ls(nodes)
+            shapes = cmds.ls(nodes, shapes=True)
+            ox_node = cmds.ls(nodes, type="BakedHairNode")
```

Comment:
@moonyuet if you "Import As" using "Guides" are the nodes still baked hair nodes?

Reply:
It could be happened sometimes.

Reply:
Not sure I understand what you mean? :)

Reply:
implement the fix [ea2736f](https://github.com/ynput/ayon-maya/pull/1/commits/ea2736ff41135161573ae0fe94403cfddb421a86)